app/models/gem_dependent.rb

```diff
@@ -25,7 +25,7 @@ def fetch_dependencies
       end
     end
 
-    dependencies
+    dependencies.flatten
```

Comment:
It was returning an array of arrays (one array per gem that was requested, and then wrapped up in a giant array after that) -- but this isn't what bundler.rubygems.org currently returns in production, so we need to make sure we match that 😬